volatility3/framework/automagic/linux.py

```diff
@@ -201,3 +215,227 @@ class LinuxSymbolFinder(symbol_finder.SymbolFinder):
     symbol_class = "volatility3.framework.symbols.linux.LinuxKernelIntermedSymbols"
     find_aslr = lambda cls, *args: LinuxIntelStacker.find_aslr(*args)[1]
     exclusion_list = ["mac", "windows"]
+
+
+class LinuxIntelVMCOREINFOStacker(interfaces.automagic.StackerLayerInterface):
+    stack_order = 34
+    exclusion_list = ["mac", "windows"]
+
+    @staticmethod
+    def _check_versions() -> bool:
+        """Verify the versions of the required modules"""
+
+        # Check SQlite cache version
+        sqlitecache_version_required = (1, 0, 0)
+        if not requirements.VersionRequirement.matches_required(
+            sqlitecache_version_required, symbol_cache.SqliteCache.version
+        ):
+            vollog.info(
+                "SQLiteCache version not suitable: required %s found %s",
+                sqlitecache_version_required,
+                symbol_cache.SqliteCache.version,
+            )
+            return False
+
+        # Check VMCOREINFO API version
+        vmcoreinfo_version_required = (1, 0, 0)
+        if not requirements.VersionRequirement.matches_required(
+            vmcoreinfo_version_required, linux.VMCoreInfo._version
+        ):
+            vollog.info(
+                "VMCOREINFO version not suitable: required %s found %s",
+                vmcoreinfo_version_required,
+                linux.VMCoreInfo._version,
+            )
+            return False
+
+        return True
+
+    @classmethod
+    def stack(
+        cls,
+        context: interfaces.context.ContextInterface,
+        layer_name: str,
+        progress_callback: constants.ProgressCallback = None,
+    ) -> Optional[interfaces.layers.DataLayerInterface]:
+        """Attempts to identify linux within this layer."""
+
+        # Verify the versions of the required modules
+        if not cls._check_versions():
+            return None
+
+        # Bail out by default unless we can stack properly
+        layer = context.layers[layer_name]
+
+        # Never stack on top of an intel layer
+        # FIXME: Find a way to improve this check
+        if isinstance(layer, intel.Intel):
+            return None
+
+        identifiers_path = os.path.join(
+            constants.CACHE_PATH, constants.IDENTIFIERS_FILENAME
+        )
+        sqlite_cache = symbol_cache.SqliteCache(identifiers_path)
```

Comment:
hm.. I was mirroring what LinuxIntelStacker does since both stackers call symbol_cache.SqliteCache()

Reply:
Ok, well, that shouldn't unless it's making SQL specific requests (which it shouldn't).  We should probably get that fixed up (in a different PR) if you want to file a ticket about it please?

Reply:
Hmmm, I see, it's because we're loading an identifiers file of a particular format...  5:S  Looks like I'll need to add something that globally loads a symbol cache...

Reply:
Ok, #1363 should take care of this now (if you review it and are happy with it, I'll fast track it rather than the couple of days wait).

Reply:
Could you please convert this to use `load_cache_manager()` please?